src/Countries/Vietnam.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+namespace Spatie\Holidays\Countries;
+
+use Carbon\CarbonImmutable;
+use DateTimeZone;
+use IntlDateFormatter;
+use Spatie\Holidays\Calendars\ChineseCalendar;
+
+class Vietnam extends Country
+{
+    use ChineseCalendar;
+
+    public function countryCode(): string
+    {
+        return 'vi';
+    }
+
+    protected function allHolidays(int $year): array
+    {
+        return array_merge([
+            'New Year\'s day' => '01-01',
+            'Reunification Day' => '04-30',
+            'Labour Day' => '05-01',
+            'Independence Day' => '09-02',
```

Comment:
That is Vietnamese holidays. Please use the Vietnamese language for labeling them.